### PR TITLE
[EDU-4288] feat: add guide - Next Pages + Middleware

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4288]

### Changes

- Add new how-to guide EN and PT- Next Pages MIddleware template

### Additional links

n/a.

[EDU-4288]: https://aziontech.atlassian.net/browse/EDU-4288?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/guides/marketplace/templates/next-pages-middleware.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+1. Add the Azion personal token to the *secrets*:
+- Read [how to generate an Azion personal token](/en/documentation/products/guides/personal-tokens/) in the documentation.
+
+```bash
+    AZION_PERSONAL_TOKEN=<value>
+```
+
+2. Add the environments for use in the action workflow in the **main.yml** file, included in the **.github/workflows** folder of your repository:
+
+```yml
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/marketplace/templates/next-pages-middleware.mdx:127]:
```suggestion
```yml title="main.yml"
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+1. Adicione o Azion personal token *secrets*:
+- Leia como [gerar um personal token da Azion](/pt-br/documentacao/produtos/guias/personal-tokens/) na documentação.
+
+```bash
+    AZION_PERSONAL_TOKEN=<value>
+```
+
+2. Adicione os ambientes para uso no action workflow no arquivo **main.yml**, dentro da pasta **.github/workflows** do seu repositório::
+
+```yml
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx:129]:
```suggestion
```yml title="main.yml"
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+Durante a implantação, você poderá acompanhar o processo através de uma janela mostrando os logs. Quando estiver concluída, uma nova página será exibida, confirmando que a edge application foi criada com sucesso.
+
+Esta página mostra informações sobre a aplicação e algumas opções para continuar sua jornada.
+
+:::note
+O link para sua edge application permite que você veja como ela fica no navegador. No entanto, leva um certo tempo para propagar e configurá-la nas edge locations da Azion. Pode ser necessário aguardar alguns minutos para que a URL seja ativada e para que a página da aplicação seja efetivamente exibida no navegador.
+:::
+
+Agora você pode [gerenciar e ajustar as configurações](#gerencie-o-template) através do Azion Console.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx:91]:
```suggestion
Agora, você pode [gerenciar e ajustar as configurações](#gerencie-o-template) através do Azion Console.
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+Se estes módulos estiverem ativados, a execução deste template pode gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para mais informações.
+:::
+
+---
+
+## Obtenha o template
+
+Para usar este template, acesse o [Azion Console](https://console.azion.com/) e clique no botão **+ Create** na página inicial.
+
+Isso abrirá um modal onde você pode selecionar **Templates** > ** [NAME]** card.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx:63]: fiquei na dúvida se ainda não tem o name ou se passou

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+Consulte a documentação sobre o [gerenciamento de edge applications](/pt-br/documentacao/produtos/build/edge-application/primeiros-passos/) para obter mais detalhes.
+:::
+
+### Continuous deployment
+
+Uma vez que o template é implantado, você pode editar e atualizar seus configurações, bem como implementar um workflow de continuous deployment, ou implantação contínua. No entanto, você precisará primeiro *declarar seus secrets no repositório do GitHub do seu projeto* para completar um segundo build com as alterações.
+
+Assim que esta segunda compilação for concluída, você poderá gerenciar seu projeto com um fluxo de trabalho de implantação contínua e editar conforme desejado.
+
+Para fazer isso, abra seu novo repositório ** [NAME]** no GitHub. Em seguida, vá para **Settings** > **Secrets and variables** > **Action** para [adicionar suas variáveis](https://docs.github.com/en/actions/security-guides/encrypted-secrets) seguindo estas instruções:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx:118]: mesma coisa do name

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx (hunk 4 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx:63]:
```suggestion
Isso abrirá um modal onde você pode selecionar **Templates** > **Next.js Pages + Middleware** card.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx (hunk 5 above)
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx:118]:
```suggestion
Para fazer isso, abra seu novo repositório **Next.js Pages + Middleware** no GitHub. Em seguida, vá para **Settings** > **Secrets and variables** > **Action** para [adicionar suas variáveis](https://docs.github.com/en/actions/security-guides/encrypted-secrets) seguindo estas instruções:
```

**Pull request merged**